pkgs/development/tools/misc/prelink/default.nix

```diff
@@ -1,20 +1,37 @@
-{ lib, stdenv, fetchurl, libelf }:
+{ stdenv, lib, fetchgit, autoreconfHook
+, libelf, libiberty
+}:
 
 stdenv.mkDerivation rec {
   pname = "prelink";
-  version = "20130503";
+  version = "2019-6-24-" + lib.substring 0 7 src.rev;
+
+  src = fetchgit {
+    url = "https://git.yoctoproject.org/git/prelink-cross";
+    branchName = "cross_prelink";
+    rev = "f9975537dbfd9ade0fc813bd5cf5fcbe41753a37";
+    sha256 = "15x1p6x9wndbjqaajhmg5nd4rcg805blixwm0l0jaiqbi9kfiprv";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+  ];
 
   buildInputs = [
-    libelf stdenv.cc.libc (lib.getOutput "static" stdenv.cc.libc)
+    stdenv.cc.libc (lib.getOutput "static" stdenv.cc.libc)
+    libelf libiberty
   ];
 
-  src = fetchurl {
-    url = "https://people.redhat.com/jakub/prelink/prelink-${version}.tar.bz2";
-    sha256 = "1w20f6ilqrz8ca51qhrn1n13h7q1r34k09g33d6l2vwvbrhcffb3";
-  };
+  # There are some failures to investigate
+  doCheck = false;
+
+  preCheck = ''
+    patchShebangs --build testsuite
+  '';
 
   meta = {
-    homepage = "https://people.redhat.com/jakub/prelink/";
+    homepage = "https://wiki.yoctoproject.org/wiki/Cross-Prelink";
+    #homepage = "https://people.redhat.com/jakub/prelink/";
```

Comment:
Thats just the old download page

```suggestion
```

Reply:
for now i just copied the prelink from ericson's pr. i'll clean it up later